cli/src/commands/bookmark/list.rs

```diff
@@ -120,6 +121,9 @@ pub struct BookmarkListArgs {
     /// Suffix the key with `-` to sort in descending order of the value (e.g.
     /// `--sort name-`). Note that when using multiple keys, the first key is
     /// the most significant.
+    ///
+    /// Configuration `ui.bookmark-list-sort-keys` is ignored when using this
+    /// option.
```

Comment:
nit: just say it is the default?

"This defaults to the `ui.bookmark-list-sort-keys` setting." (copied from `jj git push`)